src/components/DonationForm/SubComponents/SubmitButtons/MainDonationFormButtonMultiStep.vue

```diff
@@ -5,6 +5,12 @@
 </template>
 
 <script setup lang="ts">
+/**
+ * This Button handels displaying different labels based on
```

Comment:
```suggestion
 * This Button handles displaying different labels based on
```
a typo that repeats also in other components, plz search-and-replace